index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Singolo</title>
+    <link rel="stylesheet" href="my_css.css">
```

Comment:
Please give the more suitable name of css file. For example, main.css etc. Please take a look https://www.lifewire.com/naming-css-style-sheet-files-3466881